scripts/build.py

```diff
@@ -38,11 +38,20 @@
                     'armv7-android', 'arm64-android',
                     'js-web', 'wasm-web']
 
+sys.dont_write_bytecode = True
+try:
+    import build_nx64
+    sys.modules['build_private'] = build_nx64
```

Comment:
Needs fixing on the ps4 repo (which is the point of this PR, to update the PS4 repo as well)